static/elements/chromedash-form-field.js

```diff
@@ -33,22 +32,39 @@ export class ChromedashFormField extends LitElement {
     const helpText = fieldProps.help_text || '';
     const extraHelpText = fieldProps.extra_help || '';
     const type = fieldProps.type;
+    const choices = fieldProps.choices;
 
     // If type is checkbox, then generate locally.
     let fieldHTML = '';
     if (type === 'checkbox') {
       fieldHTML = html`
-          <sl-checkbox
-            name="${this.name}"
-            id="id_${this.name}"
-            size="small"
-            ?checked=${ this.value === 'True' ? true : false }
-            ?disabled=${ this.disabled }
-            >
+        <sl-checkbox
+          name="${this.name}"
+          id="id_${this.name}"
+          size="small"
+          ?checked=${this.value === 'True' ? true : false}
+          ?disabled=${this.disabled}
+        >
           ${label}
         </sl-checkbox>
-        `;
+      `;
       this.generateFieldLocally = true;
+    } else if (type === 'select') {
+      fieldHTML = html`
+        <sl-select
+          name="${this.name}"
+          id="id_${this.name}"
+          value="${this.value}"
+          size="small"
+          ?disabled=${this.disabled}
+        >
+          ${Object.values(choices).map(
+            ([index, label]) => html`
```

Comment:
Naming the first value `index` is a little confusing because it seems like it would be the array index that can be used in `map` callbacks.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

It would be better to name it `intValue`, `itemValue`, or something.

Reply:
It is generically any value, but intValue is fine.